.github/CODEOWNERS

```diff
@@ -14,6 +14,3 @@
 
 # Global owners, will be the owners for everything in the repo. Membership is tracked via https://github.com/open-telemetry/community/blob/main/community-members.md#specifications-and-proto
 *   @open-telemetry/specs-approvers
-
-# Owners for the semantic conventions generator tool
-/semantic-conventions/      @open-telemetry/specs-approvers @open-telemetry/specs-semconv-approvers @open-telemetry/specs-semconv-maintainers
```

Comment:
just adding a reminder here to remove specs-semconv-approvers and specs-semconv-maintainers from this repo permissions after merging

Reply:
done